src/wakeflow/grid.py

```diff
@@ -339,20 +349,21 @@ def _add_linear_perturbations(self, LinearPerts : _LinearPerts, rho_background :
             unperturbed density from Grid object where make_keplerian_disk has been used.
         """
 
-        # box size (in units of Hill radius), note for conversions that self.p.l = 1 Hill radius in cgs
-        x_box_size_l = 2 * self.p.scale_box_l
-        x_box_size_r = 2 * self.p.scale_box_r
-        y_box_size_t = 2 * self.p.scale_box_ang_t
-        y_box_size_b = 2 * self.p.scale_box_ang_b
-
-        min_r = self.p.r_planet - x_box_size_l * self.p.l
-        max_r = self.p.r_planet + x_box_size_r * self.p.l
-
-        min_y = -y_box_size_b * self.p.l
-        max_y =  y_box_size_t * self.p.l
+        # get linear solution           
+        lp = LinearPerts
 
-        max_phi =  np.pi / 2
-        min_phi = -np.pi / 2
+        # segment radial size (in units of Hill radius), note for conversions that self.p.l = 1 Hill radius in cgs. bsl/r stands for box_size_left/right
```

Comment:
I understand wanting short variable names but generally less confusing but longer names are preferred (not just by me but by the [python style guide](https://peps.python.org/pep-0008/). Obviously in physics sometimes we will not want to use words to represent things that are commonly represented by symbols (like `x` or `r`), but I think in this case it should definitely be `r_box_size_left` and `r_box_size_right`. This applies to some of the other variables you have named like `phi_bst`, please change to `phi_box_size_top`. We wish to make the code as _readable_ as humanly possible.

There may be other places in `wakeflow` that use short and unreadable variable names, which I should definitely change! Some parts are getting pretty old at this point and I used to be less fussy haha 😄  (and I knew less about coding)